examples/terraform-gcp-ig-example/main.tf

```diff
@@ -0,0 +1,43 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A REGIONAL MANAGED INSTANCE GROUP
+# See test/terraform_gcp_ig_example_test.go for how to write automated tests for this code.
+# ---------------------------------------------------------------------------------------------------------------------
+
+# Create a Regional Managed Instance Group
+resource "google_compute_region_instance_group_manager" "example" {
+  project = "${var.gcp_project_id}"
+  region = "${var.gcp_region}"
+
+  name = "${var.cluster_name}-ig"
+  base_instance_name = "${var.cluster_name}"
+  instance_template = "${google_compute_instance_template.example.self_link}"
+
+  target_size = "${var.cluster_size}"
+}
```

Comment:
Please run `terraform fmt`

Reply:
Done in dc29013d3611839cc35ec1d06d3d899c477372f9.